velox/dwio/text/writer/BufferedWriter.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/dwio/text/writer/BufferedWriter.h"
+
+namespace facebook::velox::text {
+
+BufferedWriter::BufferedWriter(
+    std::unique_ptr<dwio::common::FileSink> sink,
+    std::shared_ptr<memory::MemoryPool> pool)
```

Comment:
add `&&` and `const &`, respectively, to avoid object copies.

Reply:
It was written this way to be more consistent with our codebase

https://github.com/facebookincubator/velox/blob/c04652462250469d759396aac9d68d441ac1eb2a/velox/dwio/parquet/writer/Writer.cpp#L41
https://github.com/facebookincubator/velox/blob/c04652462250469d759396aac9d68d441ac1eb2a/velox/dwio/parquet/writer/Writer.cpp#L221